GLMakie/src/GLAbstraction/GLTexture.jl

```diff
@@ -134,7 +134,7 @@ So Array{Real, 2} == Texture2D with 1D Colorant dimension
 Array{Vec1/2/3/4, 2} == Texture2D with 1/2/3/4D Colorant dimension
 Colors from Colors.jl should mostly work as well
 """
-Texture(image::Array{T, NDim}; kw_args...) where {T <: GLArrayEltypes, NDim} =
+Texture(image::AbstractArray{T, NDim}; kw_args...) where {T <: GLArrayEltypes, NDim} =
```

Comment:
That's not correct...
`pointer(...)` is not guaranteed to be correctly implemented for all AbstractArrays, and might even return a garbage pointer if used on the wrong array type. A lot of AbstractArrays like view + adjoint even have pointer defined, which just point to the parent array, which is of course not what you want from a view/mapping.

You'll need something like `Texture(image::AbstractArray{T, NDim}) = Texture(convert(Array{T, NDIM}, image))` to be on the safe side.
I'm guessing this "fixed" your issue but the data must have been incorrectly permuted:
<img width="442" alt="image" src="https://user-images.githubusercontent.com/1010467/214568085-1eaa9a81-49a9-41ec-a9e1-ba5d7067e58a.png">